PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -377,6 +377,7 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
         ecalEnergy += icand->ecalEnergy();
         hcalEnergy += icand->hcalEnergy();		
 	sumEtaTimesEnergy += icand->positionAtECALEntrance().eta()*icand->energy();
+	sumPhiTimesEnergy += icand->positionAtECALEntrance().phi()*icand->energy();
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.